stko/molecular/networkx/network.py

```diff
@@ -0,0 +1,64 @@
+"""
+Network
+=======
+
+Class for defining a networkx graph from a molecule.
+
+"""
+
+import logging
+import networkx as nx
+
+logger = logging.getLogger(__name__)
+
+
+class Network:
```

Comment:
I would suggest making it immutable and having the methods which modify the instance return modified clones instead.

Reply:
Always agreed!